runtime/isp_run_app.py

```diff
@@ -240,6 +230,16 @@ def main():
     isp_utils.removeIfExists(run_dir)
     isp_utils.doMkDir(run_dir)
 
+    # use exiting policy directory if -p arg refers to path
+    if (len(policies) == 1 and 
+        "/" in args.policies[0] and
+        os.path.isdir(policies[0])):
+        policy_dir = os.path.abspath(policies[0])
+        policy_name = os.path.basename(policy_dir)
+    else:
+        policy_name = isp_utils.getPolicyFullName(policies, args.global_policies, args.policy_debug)
+        policy_dir = os.path.join(run_dir, policy_name)
```

Comment:
Related to the previous comment, maybe policies should be a subdirectory of the run_dir.

Reply:
`isp_install_policy` already creates that directory.  As an aside, are you sure you're commenting on the right commit?  This code should look a little different and be about 20 lines higher up in the file.